README.md

```diff
@@ -1,2 +1,228 @@
-# example-application
-Example out-of-tree application that is also a module
+![Zephyr Project logo](https://github.com/zephyrproject-rtos/zephyr/blob/v1.14-branch/doc/images/Zephyr-Project.png)
```

Comment:
Probably should describe what is in this repository.

Reply:
@cfriedt The description is in the Getting Started section.  Perhaps it needs to be above it?

Reply:
> @cfriedt The description is in the Getting Started section. Perhaps it needs to be above it?

I think the document should be reorganized to have a quickstart for the impatient front and center. Something people can just start copy/pasting into their terminals right away, and that works on Windows too.

Then all the information in "Getting started", "Example vs sample", etc. can follow.

Reply:
Ah sorry, I saw the "Zephyr is.." part and assumed there was a copy-paste. Maybe best to just use a hyperlink instead of another description.

Reply:
I put a Module Overview section at the top to avoid that assumption, and moved the Scope/Design Principles/Example vs Sample sections into the Overview. This leaves Getting Started with copyable commands (for Ubuntu).

Reply:
I have also added subsections for macOS and Windows commands.